src/pages/extensions/iter8/Iter8ExperimentDetails/CriteriaInfoDescription.css

```diff
@@ -0,0 +1,3 @@
+.headertoggle .pf-c-button.pf-m-plain {
```

Comment:
I think we don't commit directly the *.css file but the *.scss file.
Those are compiled in build time.

And yes, code imports the resulting "css" one but not the scss.
So, I'd just rename this to scss and allow that the resulting css is part of the local file but not part of the commit.

Under the pages/ServicesDetails/ServiceInfo there are several files that can help as examples, like ServiceDescription.tsx.

Also, we tend to add inline style at code and avoid to use the scss/css ones as those tend to build it together and sometimes it may introduce some side effect in other components that may use same style classes.